recipes/code-aster/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "aster" %}
+{% set version = "14.6.0" %}
+#{% set sha256 = "2afa430a588dc690c7797e3b87de67742cd19a92c82bcb715b007709b0983131" %}
+{% set sha256 = "dcb3900de1e5c0624060253660a63512ab415cbe143cdd87867a0276ef6bf233" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  #url: https://www.code-aster.org/FICHIERS/aster-full-src-14.6.0-1.noarch.tar.gz # Downloaded bytes did not match Content-Length
+  url: https://collegamenti.s3.eu-west-1.amazonaws.com/aster-full-src-{{ version }}-1.noarch.tar.gz
```

Comment:
How did you find this url? It doesn't seem like an official release.

Reply:
I could have used (and would have liked to) the official release url (the commented one), this is where I downloaded the software. The main issue is that when using build-locally it did not download in a stable way : the package is quite big, the download timeouts are a bit too strict. So I downloaded it and uploaded it into s3 to make it stable. If you have any way to suggest that I can use to make sure that it is downloaded safely I gladly get rid of the s3 link ?

Reply:
I just tested it again : the exact error message is "conda.CondaError: Downloaded bytes did not match Content-Length". Of course, when downloaded manually, the hash is the same as provided in alternative link.

Reply:
I provided the URL for direct download from their official sourceforge

Reply:
Thanks for this :-) I found the new official link (the project moved from sourceforge to gitlab) and used that one